docs/stdlib.md

```diff
@@ -1230,6 +1193,43 @@ Get the pid of the parent process
 #### Last Expression
 - **uint32**: The pid of the parent process
 
+### socket_cookie
```

Comment:
This is in the macro section. We need to update the script to support stdlib functions.